### PR TITLE
Add js

### PR DESCRIPTION
**Pull request by ManarOmari**
add nav menue when click to humburger the menue appear

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,30 +5,28 @@
   <meta charset="UTF-8" />
```
Comment (Munsa1) [inline index.html:5]: - [x] This is the first project where you are using Javascript. Kindly set up the correct linters for this project as it is a requirement. Refer to this [repository](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

File: index.html [hunk 2]
```diff
@@ -5,30 +5,28 @@
...
-          </li>
-          <li>
-            <a href="#">Contact</a>
-          </li>
-        </ul>
+          <a href="#" class="nav-brand">My Logo</a>
+          <ul class="list-nav ">
+            <li class="nav-item"><a href="#" class="nav-link">Portfolio</a></li>
+              <li class="nav-item"><a href="#"class="nav-link">About</a></li>
+              <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
```
Comment (Munsa1) [inline index.html]: - [x] These links that come up when the menu bar is clicked should link to the appropriate section of the webpage. For example the `Portfolio` link should link to the top of the index page, the `About` should go to the about section when clicked and the `Contact` should go to the contact section when clicked. Kindly fix this issue as it is part of the requirements.
image

File: index.html [hunk 3]
```diff
@@ -5,30 +5,28 @@
...
+            </ul>
+            <div class="hamburger">
+             <span class="bar"></span>
+             <span class="bar"></span>
+             <span class="bar"></span>
+            </div>
```
Comment (Munsa1) [inline index.html:27]: - [ ] Kindly follow the design template in Figma, including colors, fonts and background images, take a look at the difference between your design and the provided one.

Figma
image

Yours
image

- The background color should cover the whole mobile view and the `X` button should be positioned like in the design, so should the links be as well. Also pay attention to the font of the links.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @ManarOmari @Suzi216, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

- [x] Add commits from both paries, I only see commits from one end. Please this is a requirement
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ManarOmari) [reply index.html]: solved

File: index.html (hunk 1 above)
Comment (ManarOmari) [reply index.html:5]: solved

Comment (ManarOmari) [reply index.html:5]: solved

File: index.html (hunk 2 above)
Comment (ManarOmari) [reply index.html]: solved

File: index.html (hunk 3 above)
Comment (ManarOmari) [reply index.html:27]: solved

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @ManarOmari 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**